core/src/main/java/com/devonfw/tools/solicitor/common/ResourceToFileCopier.java

```diff
@@ -205,10 +205,10 @@ public String copyReourcesToFile(ResourceGroup resourceGroup, String targetDir)
         switch (resourceGroup) {
         case USERGUIDE:
             new CopySequenceBuilder()
-                    .withCopyOperation("classpath:solicitor_userguide.pdf", "target/solicitor_userguide.pdf")
+                    .withCopyOperation("classpath:master-solicitor.pdf", "target/master-solicitor.pdf")
                     .withCopyOperation("classpath:solicitor_licenseinfo.html", "target/solicitor_licenseinfo.html")
                     .replaceInTarget("target", targetDir).execute();
-            returnString = "solicitor_userguide.pdf";
+            returnString = "master-solicitor.pdf";
```

Comment:
I think the filename does not have to be changed. IMHO `solicitor_userguide` makes much more sense than `master-solicitor`.

Reply:
Agreed. just to align with rest of the devon document repos - had to make it master-solicitor.

Reply:
I think there is actually no reason for that. None of our algorithms rely on that behavior to have this specific name.